app/views/curation_concerns/base/_representative_media.html.erb

```diff
@@ -1,5 +1,5 @@
-<% if presenter.representative_id.present? && (fs = ::FileSet.find(presenter.representative_id)) %>
-  <%= media_display fs %>
+<% if presenter.representative_id.present? %>
```

Comment:
@jeremyf I hesitate to move the HTML into the helper.  it is harder for the UI people if the HTML is sprinkled in the helpers, presenters. I guess we could move the Image tag into it's own partial, or check to see if there is already another one and then just render that partial.

 I agree that the code is duplicated, but it is called from multiple places.